src/openrct2/ride/ride_ratings.c

```diff
@@ -610,32 +610,71 @@ static void ride_ratings_calculate(rct_ride *ride)
 
 static void ride_ratings_calculate_value(rct_ride *ride)
 {
+    static const int age_table_new[][4] = {
```

Comment:
A struct would mean you wouldn't need to add the comments and make it more legable.
```
struct{
    int months;
    int multiplier;
    int divisor;
    int summand;
};
```